lib/measures/add_ems_emissions_reporting/measure.rb

```diff
@@ -193,11 +332,23 @@ def run(model, runner, usr_args)
     his_yr_data.each {|r| ems_prgm.addLine("SET his_yr_val = #{r[annual_historical_subregion]}") if r[0].to_i == annual_historical_year}
     ems_prgm.addLine('SET elec = Ele_Sen')
     ems_prgm.addLine('SET conv = 1000000 * 60 * 60') # J to MWh (1000000J/MJ * 60hr/min * 60 min/sec)
-    ems_prgm.addLine('SET fut_hr = fut_hr_val * elec / conv')
-    ems_prgm.addLine('SET his_hr = his_hr_val * elec / conv')
-    ems_prgm.addLine('SET fut_yr = fut_yr_val * elec / conv')
-    ems_prgm.addLine('SET his_yr = his_yr_val * elec / conv')
-
+    ems_prgm.addLine('SET conv_kg_mt = 0.001') # kg to metric ton
```

Comment:
@nllong is metric tons okay for what you're doing

Reply:
yes, we need to change to metric tons!

Reply:
oh, even better! :grin: